apps/storybook/src/Zoom.stories.tsx

```diff
@@ -49,4 +18,43 @@ export default {
       options: ['Alt', 'Control', 'Shift'],
     },
   },
-} as Meta<ZoomProps>;
+} satisfies Meta<typeof Zoom>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
+  render: (args) => {
+    return (
+      <VisCanvas
+        abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
+        ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
+      >
+        <Pan />
+        <Zoom {...args} />
+        <ResetZoomButton />
+      </VisCanvas>
+    );
+  },
+} satisfies Story;
+
+export const ModifierKey = {
+  ...Default,
+  args: {
+    modifierKey: ['Control'],
+  },
+} satisfies Story;
+
+export const MultipleModifierKeys = {
+  ...Default,
+  args: {
+    modifierKey: ['Control', 'Shift', 'Alt'],
```

Comment:
I don't think it is worth having all three keys: it is a bit janky to test. Let's remove one.